desktop-widgets/printoptions.cpp

```diff
@@ -47,6 +48,7 @@ void PrintOptions::setup()
 
 	connect(ui.printInColor, SIGNAL(clicked(bool)), this, SLOT(printInColorClicked(bool)));
 	connect(ui.printSelected, SIGNAL(clicked(bool)), this, SLOT(printSelectedClicked(bool)));
+	connect(ui.printLandscape, SIGNAL(clicked(bool)), this, SLOT(printLandscapeClicked(bool)));
```

Comment:
Pleased don't add new SIGNAL/SLOT-style connects. In this trivial case you might even add a lambda as in the next line.

Reply:
Turned all three into lambdas.

Reply:
Thanks!

Reply:
yay... more... ƛ - just what I always wanted (just joking... this is the right thing to do here)